snuba/utils/health_info.py

```diff
@@ -120,17 +125,52 @@ def get_health_info(thorough: Union[bool, str]) -> HealthInfo:
     )
 
 
-def filter_checked_storages() -> List[Storage]:
-    filtered_storages: List[Storage] = []
+def filter_checked_storages(filtered_storages: List[Storage]) -> None:
     all_storage_keys = get_all_storage_keys()
     for storage_key in all_storage_keys:
         storage = get_storage(storage_key)
         if storage.get_readiness_state().value in settings.SUPPORTED_STATES:
             filtered_storages.append(storage)
-    return filtered_storages
 
 
-def check_clickhouse(metric_tags: dict[str, Any] | None = None) -> bool:
+def sanity_check_clickhouse_connections() -> bool:
+    """
+    Check if at least a sinlge clickhouse query node is operable,
+    returns True if so, False otherwise.
+    """
+    setup_logging(None)
+    logger = logging.getLogger("snuba.health")
+
+    storages: List[Storage] = []
+
+    try:
+        filter_checked_storages(storages)
+    except KeyError:
+        pass
```

Comment:
I'm not entirely sure if I should use the checked_storages function to find clickhouse connections. Please let me know if there's a better idea.

Reply:
I think this is correct